### DISCUSSION TITLE
Add a button to fold/expand all actions in the Input Map

### DISCUSSION THREAD

**Discussion by ArnaudUNC** [godotengine/godot-proposals#7884] [Editor]
For a better overview of all the actions when there are many in a project.

Example: go from this ↓
![image](https://github.com/godotengine/godot-proposals/assets/64565072/132789de-01ab-4625-a367-612cd52aef88)

To this ↓ (and vice versa) with a single button click
![image](https://github.com/godotengine/godot-proposals/assets/64565072/3fe17b05-ef4d-463a-8782-d14c17bcd1b1)

**Comment by aXu-AP** [comment]
This is a good functionality, altough as an alternative it could be triggered by ctrl + open/close one of the items. Similiar shortcut is used at least in Photoshop to open/close all layer groups (altough there it's ctrl+alt+click).

**Reply by timothyqiu** [reply]
Ctrl+Click sometimes also mean revealing the current branch. So we probably need a switch on Tree.

**Comment by Mickeon** [comment]
I feel like https://github.com/godotengine/godot-proposals/issues/4910 would intrinsically solve this without requiring additional buttons/shortcuts.
